libnavigation-core/src/test/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetryTest.kt

```diff
@@ -193,11 +213,20 @@ class MapboxNavigationTelemetryTest {
         every {
             mapboxNavigation.registerRoutesObserver(capture(routesObserverSlot))
         } just runs
+
+        every {
+            globalUserFeedbackObserver.onNewUserFeedback(capture(globalUserFeedbackSlot))
+        } just runs
+
+        every {
+            localUserFeedbackObserver.onNewUserFeedback(capture(localUserFeedbackSlot))
+        } just runs
     }
 
     @After
     fun cleanUp() {
         unmockkObject(MapboxMetricsReporter)
+        unmockkStatic(TelemetryUtils::obtainUniversalUniqueIdentifier)
```

Comment:
Haven't you come up with a custom rule in order to reduce test code yet? For one static mock it would be one line against 8 if you need to create Before and After methods and here they're cerated specifically to unmock things so they wouldn't be present otherwise.

Reply:
We don't have such a rule. By the way, we recently had a discussion where it was agreed to write less code in `@Before` and `@After` methods, because sometimes logic in these methods is not valid for all of the tests. With `@Rule`s with have the same problem.

Reply:
Well, yes, but sometimes you do write code in Before and After and with Rule you wouldn't have to write these methods at all. Anyhoo, it's just an observation. We had it on our previous project and I found it quite convenient.